src/Microsoft.FeatureManagement.AspNetCore/FeatureFlagsEndpointFilterExtensions.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.AspNetCore.Builder;
```

Comment:
The new files are missing the [license header](https://github.com/microsoft/FeatureManagement-Dotnet/blob/76969fa89125d1075a25980081365a522966616e/src/Microsoft.FeatureManagement/ConfigurationWrapper.cs#L1).